spark-jobs/src/main/scala/filodb.downsampler/BatchDownsampler.scala

```diff
@@ -14,10 +14,11 @@ import filodb.cassandra.FiloSessionProvider
 import filodb.cassandra.columnstore.CassandraColumnStore
 import filodb.core.{DatasetRef, ErrorResponse, Instance}
 import filodb.core.binaryrecord2.RecordSchema
-import filodb.core.downsample.{ChunkDownsampler, DoubleChunkDownsampler, HistChunkDownsampler, TimeChunkDownsampler}
+import filodb.core.downsample._
```

Comment:
I generally prefer to see this change going the other way -- fewer wildcards. I don't care too much, however.

Reply:
It was intellij auto imports at play. It introduces wildcard after a certain number of whitelisted imports.